drivers_experimental_linux.go

```diff
@@ -1,9 +1,5 @@
 package libnetwork
 
-import "github.com/docker/libnetwork/drivers/ipvlan"
-
 func additionalDrivers() []initializer {
```

Comment:
Perhaps best to do that in a separate commit; looks like there's quite some dead code and docs we can remove then https://github.com/docker/libnetwork/search?utf8=✓&q=experimental&type=

Reply:
+1

Reply:
@thaJeztah @abhi though I agree that promoting IPVLAN driver will result in NOOP code wrt experimental drivers, I still believe it is useful to retain it instead of removing these files altogether. This acts as a template for anyone willing to add experimental drivers. I would hate to see folks spending more than necessary to add the infra to add experimental drivers.

Reply:
That's what we have version control for; better to remove dead code than keeping it "just in case"

Reply:
And the commit that removes it can be used as template (that's why it's good to keep it in a single, separate commit)

Reply:
@thaJeztah sure. But anyone in the future who is trying to add a new experimental driver may not know this history and may not dig into the git history to identify this particular commit. They would have spent valuable time redoing the work and we may have to point them back to this commit, etc. Having framework code in place is useful in certain situations. 

IMO Code that performs a logic or functional in nature being dead will confuse everyone, while framework code will help guide folks. In any case, if other maintainers prefer to remove this unused code (which it seems like), I will go ahead and remove it.

Reply:
> But anyone in the future who is trying to add a new experimental driver may not know this history and may not dig into the git history to identify this particular commit.

Shouldn't someone who is adding a new new experimental driver be talking to the community up front? If so then at that point they can be pointed at the history/commit. If not, well, that's why we suggest that people discuss big things before doing lots of work...